docs/content/releases.md

```diff
@@ -15,6 +16,60 @@ We recommend against:
 2. Building new custom FIPS images.
 {{< /note >}}
 
+## 3.7.0
+
+30 Sept 2024
+
```

Comment:
Added support for VirtualServer & TransportServer to listen on a specific IP when configuring a `listener`, allowing NGINX to bind to a specific interface.
Allow an End Session Endpoint to be configured for OIDC providers, allowing a user to fully logout. This change also adds support for configuring a post-logout redirect URI, allowing a users to be redirected to a custom logout page.
The `access_log` directive can now be configured to point to a syslog log server. Perviously, access logs defaulted to standard out. This change allows for log parsers aggregators such as Splunk to ingest access logs from NGNIX.
Each installation of NGINX Ingress Controller will now create a unique lease object, allowing for multiple deployment of NGINX Ingress Controller when leader election is enabled.